src/.npmignore

```diff
@@ -1,4 +1,7 @@
 *.map
 *.ts
 !*.d.ts
-tsconfig.json 
\ No newline at end of file
+tsconfig.json 
+/platforms/android/**/*
+!platforms/android/include.gradle
```

Comment:
I think include.gradle should not be ignored. Check point 5: https://github.com/NativeScript/nativescript-plugin-seed/issues/65